blocks/library/heading/index.js

```diff
@@ -29,6 +29,8 @@ registerBlockType( 'core/heading', {
 
 	className: false,
 
+	supportAnchor: true,
```

Comment:
I personally think the block alignments should be part of these generic attributes. And I don't have a strong preference on how to declare support for these attributes.